operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -520,6 +528,7 @@ public void createRenewOrReplace(String namespace, String clusterName, Map<Strin
         if (!generateCa) {
             certData = caCertSecret != null ? caCertSecret.getData() : emptyMap();
             keyData = caKeySecret != null ? singletonMap(CA_KEY, caKeySecret.getData().get(CA_KEY)) : emptyMap();
+            renewalType = isCaCertThumbprintChanged() ? RenewalType.REPLACE_KEY : RenewalType.NOOP;
```

Comment:
Can we not use the policy defined in the CR?

Reply:
We actually don't know what the user will do, if generating the new CA using the same key or using a new key. Setting the more strict policy (so REPLACE_KEY) means that we should get enough rolling of our pods for the overall renewal process. We are "blind" here, so it's playing safer this way.